packages/ingest-mongodb-public/ingestion_guide.md

```diff
@@ -0,0 +1,75 @@
+
+Meeting with the person requesting the ingestion is the best way to get this information and reach alignment.
+
+### Meeting Questions
+
+In the meeting cover the following questions:
+
+#### 1. What is the goal for ingesting the content? 
+
+Get a baseline understanding of not just _what_ they want ingested but _why_. This can help guide the rest of the converastion.
+
+Also, sometimes we do not need to ingest additional content to achieve a specific goal.
+
+#### 2. Where's the content located?
+
+Is it in a CMS, a Github repo, on the web, etc.?
+
+Generally, ingesting from a non-web source like a repo or CMS works better. It can be more upfront work, but results in cleaner data and quicker ingestion run times (scraping the web is slow and error-prone). 
+
+The ideal state is that the content exists in clean Markdown/text with a link to the URL where the content can be found on the web. Try to get as close as possible to this.
+
+#### 3. What answers do you expect to recieve from the chatbot based on the ingested content?
+
+If there's any complexity regarding the ingestion, you should create an evaluation dataset for the topics covered in the new content.
+
+The requestor can create the example dataset in a Google Sheet following the [template of this sheet of evaluation questions for ingesting website content](https://docs.google.com/spreadsheets/d/1khZ9jwgffIUb-bDXqnem5o1Xq3RZ1sz8_JXFc3o9GCE/edit?pli=1&gid=0#gid=0).
+
+The single most important part of the evaluation dataset is the queries. These should be well thought through and broadly representative of what a user might ask. The provided links are also very useful, as they help us coorelate the answers back to specific pieces of content. The sample answer is useful for understanding the intent of the query and can be used as a 'reference answer' in LLM-as-a-judge evaluations.
+
+#### 4. Are there any specific questions that you want the chatbot to answer with pre-written answers?
+
+See if there are any pre-written question/answer/link sets that they want to include. We can add these as verified answers. 
+
+Generally, verified answers should be used sparingly. Ideally, the chatbot can just answer using RAG. Verified answers are more of a 'patch'.
+
+You can add verified answer in the [verified-answers.yaml](https://github.com/mongodb/chatbot/blob/main/verified-answers.yaml) file.
+
+## Data Ingestion Checklist
+
+When ingesting new content, you want to perform the following actions:
+
+- [ ] Create the data source for the content
+- [ ] Add data source to `ingest-mongodb-public` and run ingest in dev environment
+- [ ] Adapt the evaluation Google Sheet as eval cases in the repo. See [`conversations.eval.ts`](https://github.com/mongodb/chatbot/blob/main/packages/chatbot-server-mongodb-public/src/conversations.eval.ts) as reference.
```

Comment:
For the future, might be nice to set up a script that creates `.eval` files in one shot from the google sheet

Reply:
yea, or translates the sheet to a yaml file.